src/Sylius/Bundle/ApiBundle/Resources/translations/validators.en.yaml

```
@@ -22,6 +22,9 @@ sylius:
         not_longer_available: 'The product variant with name %productVariantName% does not exist.'
         not_sufficient: 'The product variant with %productVariantCode% code does not have sufficient stock.'
         product_variant_with_name_not_sufficient: 'The product variant with %productVariantName% name does not have sufficient stock.'
+    reset_password:
+        invalid_token: 'No user found with reset token: %token%.'
```

```suggestion
        invalid_token: 'Password reset token %token% has not been found.'
```
or maybe simply
```suggestion
        invalid_token: 'Password reset token %token% is invalid.'
```
either way, it's a bit less internal logic leaked